src/arrays/practice/1.js

```diff
@@ -1,9 +1,35 @@
-function arrayManipulations() {
-  // your code here
-  let arr = [1,2,3];
-  arr.filter((elem, i) => {
-      console.log(elem);
-  });
+function arr1(arr) {
+    let LastItem = arr[arr.length - 1];
+    return LastItem;
 }
 
-arrayManipulations();
+console.log(arr1([1, 33, 5, 7, 8]));
+
+function arr2(arr) {
+    return arr[arr.length - 1];
+}
+
+console.log(arr2([1, 33, 5, 7, 8]));
+
+
+function arr3(arr, n) {
+    arr.push(n);
+    return arr;
+}
+
+console.log(arr3([1, 33, 5, 7, 8], 9));
+console.log(arr3([1, 33, 5, 7, 8], 10));
+
+
+function arr4(arr) {
```

Comment:
Make function dynamic. Function should take arrSong as input parametr

Reply:
problem solved